---
title: Software
software: True
---

## UNDR ROVER 

Unmapped primer directed read overlap variant caller.

Web: [http://bjpop.github.io/undr_rover/](http://bjpop.github.io/undr_rover/)

#### Downloads

* [Source repository on github](https://github.com/bjpop/undr_rover)

    You can download the UNDR ROVER source repository using git with this command:

    `git clone https://github.com/bjpop/undr_rover` 

#### Description 

A new variant caller for the Hi-Plex system with the same highly accurate results 
as ROVER but with improved runtime performance. 
Variants are called directly on input FASTQ files instead of aligned BAM files. The avoidance of the 
computationally costly alignment step greatly reduces the time to produce results.

****

## Hi-Plex Primer

A tool for generating primers for the Hi-Plex targeted, multiplexed DNA sequencing strategy. 

#### Downloads

* [Source repository on github](https://github.com/bjpop/hiplex-primer).

    You can download the Hi-Plex Primer source repository using git with this command:

    `git clone https://github.com/bjpop/hiplex-primer` 

#### Description 

Hi-Plex Primer generates primer pairs for a given set of target regions. See the [user guide](http://bjpop.github.io/hiplex-primer/) for information about using the tool.

****

## ROVER 

Read-pair overlap considerate variant-calling software for PCR-based massively parallel sequencing datasets.

#### Downloads

* [Source repository on github](https://github.com/bjpop/rover).

    You can download the ROVER source repository using git with this command:

    `git clone https://github.com/bjpop/rover` 

#### Description 

ROVER-PCR Variant Caller enables users to quickly and accurately identify
genetic variants from PCR-targeted, overlapping paired-end MPS datasets. The
open-source availability of the software and threshold tailorability enables
broad access for a range of PCR-MPS users.

This program is described in the paper [ROVER variant caller: read-pair overlap considerate variant-calling software
   applied to PCR-based massively parallel sequencing datasets](http://www.scfbm.org/content/9/1/3), published in Source Code for Biology and Medicine.

